contract/static/src/js/section_and_note_fields_backend.js

```diff
@@ -13,26 +13,27 @@ odoo.define("contract.section_and_note_backend", function(require) {
 
```

Comment:
I think that this issue happens because the script is loading before that the main widget script.

This change should fix it... with this you ensure that the main widget script is loaded:
```suggestion
require('account.section_and_note_backend');
```

Reply:
Thanks, I have changed it with your suggestion, is it correct now?